### PR TITLE
Upgrade context version

### PR DESCRIPTION
**Pull request by yogeshjain999**

### CODE REVIEW

File: test/dry_container_test.rb [hunk 1]
```diff
@@ -21,5 +21,5 @@ class Create < Trailblazer::Operation
   # also allows our own options PLUS containers.
   it { Create.({}, {"model" => String}, my_container)["model"].must_equal String }
   it { Create.({}, {"model" => String}, my_container)["user_repository"].must_equal Object }
-  it { Create.({}, {"user_repository" => Integer}, my_container)["user_repository"].must_equal Integer }
+  it { Create.({}, my_container, {"user_repository" => Integer})["user_repository"].must_equal Integer }
```
Comment (apotonick) [inline test/dry_container_test.rb:24]: I am still unsure if we should keep supporting that API - I'd rather require users to build a contain "chain" themselves and allow only one argument.

Comment (yogeshjain999) [reply test/dry_container_test.rb:24]: Hmm right.

**Review by apotonick — APPROVED** [review]

File: test/dry_container_test.rb (hunk 1 above)
Comment (seuros) [reply test/dry_container_test.rb]: _().must_equal

Comment (yogeshjain999) [reply test/dry_container_test.rb]: This has been removed @seuros ⚔️

**Pull request merged**